admin_guide/cluster_management.rst

```diff
@@ -464,7 +480,7 @@ To set non-sensitive libpq connection parameters to be used for all node connect
   SET citus.node_conninfo =
     'sslrootcert=/path/to/citus.crt sslmode=verify-full';
 
-There is a whitelist of parameters that the GUC accepts. See the :ref:`node_conninfo <node_conninfo>` reference for details.
+There is a whitelist of parameters that the GUC accepts. See the :ref:`node_conninfo <node_conninfo>` reference for details. By default, nodes encrypt communication with SSL (using sslmode=require).
```

Comment:
`sslmode=requrie` doesn't encrypt the traffic on its own, also `ssl` has to be set to `on` at the other end etc.
I think rewording this to sth like;

`By default, nodes require connections to be encrypted`.